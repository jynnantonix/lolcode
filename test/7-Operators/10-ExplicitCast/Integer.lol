This test checks that the explicit cast operator works correctly when casting to integer types. 

HAI 1.2
	I HAS A var
	I HAS A var2 ITZ 4
	I HAS A var3 ITZ 3.25
	I HAS A var4 ITZ "HAI MAUS"
	I HAS A var5 ITZ WIN
	VISIBLE SUM OF 0 AN NOT MAEK var2 A NUMBR MKAY?
	VISIBLE SUM OF 0 AN NOT MAEK var3 A NUMBR MKAY?
	VISIBLE SUM OF 0 AN NOT MAEK var4 A NUMBR MKAY?
	VISIBLE SUM OF 0 AN NOT MAEK var5 A NUMBR MKAY?
	VISIBLE SUM OF 0 AN NOT MAEK var A NUMBR MKAY?
KTHXBAI


