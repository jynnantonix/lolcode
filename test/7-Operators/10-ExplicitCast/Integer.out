0 
0 
Unexpected error: invalid literal for int() with base 10: 'HAI MAUS'
